src/clojure/nrepl/core.clj

```diff
@@ -221,13 +225,21 @@
 
 (add-socket-connect-method! "nrepl" {:transport-fn transport/bencode
                                      :port 7888})
+(defmethod transport->uri-scheme #'transport/bencode [_] "nrepl")
```

Comment:
I meant you could have added this to `add-socket-connect-method`, but generally that's not a big deal.